tests/Tests.elm

```diff
@@ -85,6 +87,19 @@ regressions =
             \positiveInt ->
                 positiveInt
                     |> Expect.greaterThan 0
+        , fuzz
+            (custom (Random.int 1 8) Shrink.noShrink)
+            "fuzz tests run 100 times"
+            (Expect.notEqual 5)
+            |> expectToFail
+          {- If fuzz tests actually run 100 times, then asserting that no number
+             in 1..8 equals 5 is virtually guaranteed to fail. If they only run
```

Comment:
I'd like to see some hard numbers here, or a link to our discussion on Github. Otherwise, the next contributor modifying this code (or anything related to it) has to redo the calculations.